docs/client/full-api/api/templates.md

```diff
@@ -103,7 +103,7 @@ template is rendered for the first time.
 ```
 
 ```javascript
-Template.myPictures.onRendered(function () {
+Template.myPictures.onRendered( () => {
```

Comment:
This will not work - you can't replace functions that use `this` with arrow functions. That's a common misconception.